### DISCUSSION TITLE
HelloFresh

### DISCUSSION THREAD

**Discussion by timlitw** [moralmunky/Home-Assistant-Mail-And-Packages#857] [Ideas]
what information do you need to add Hello Fresh as an option?
I can provide emails that the box has shipped
the box has been delivered
delivery update if the box is delayed
the picture of the delivered box doesn't come from HelloFresh tho it is SMS from Veho

**Comment by firstof9** [comment]
Do they use UPS/USPS/FedEx/etc to deliver the package?

**Comment by timlitw** [comment]
They use veho

**Reply by firstof9** [reply]
_Technically_ you could add this to your local copy in `const.py` and test how well it works. The submit a PR once you verified it works.

Add it after this line:

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/6abdb93cc06ab5ade630ce5ca797e0283ebb7fe0/custom_components/mail_and_packages/const.py#L373

for the email address and subject searches.

Then to make the sensor:

https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/blob/6abdb93cc06ab5ade630ce5ca797e0283ebb7fe0/custom_components/mail_and_packages/const.py#L655

Add after this line using the format of the previously listed sensors. :slightly_smiling_face:
